be/src/olap/rowset/beta_rowset_writer.cpp

```diff
@@ -743,9 +819,25 @@ RowsetSharedPtr BetaRowsetWriter::build() {
         return nullptr;
     }
 
+    if (config::enable_final_pass_segcompaction) {
+        uint64_t begin = GetCurrentTimeMicros();
```

Comment:
use OlapStopWatch instead of GetCurrentTimeMicros

Reply:
> use OlapStopWatch instead of GetCurrentTimeMicros

Thanks for your suggestion. I have checked OlapStopWatch, which is great tool but lack of flexibility coz it start timing when it is contructed. Thought I can put it into the scope here but the code are changing from time to time. I think it is more useful when do function level timing. So I would like to stick to GetCurrentTimeMicros if there is no performance issue.